app/models/garden_plant.rb

```diff
@@ -0,0 +1,6 @@
+class GardenPlant < ApplicationRecord
+  validates :planted_date, presence: true
+
```

Comment:
@hashmaster3k nice! hopefully from the user's perspective, we can allow them to add multiple records at once for the same plant vs having to enter separately for each plant.